lib/health-data-standards/export/qrda/entry_template_resolver.rb

```
@@ -53,11 +53,18 @@ def qrda_oid_for_hqmf_patient_characteristic(vs_oid, qrda_version = nil)
             '2.16.840.1.113883.10.20.24.3.101'
           when '2.16.840.1.113883.3.526.3.1189', '2.16.840.1.113883.3.526.3.1170', '2.16.840.1.113883.3.600.2390'
             # Patient Characteristic Tobacco User/Non-User
-            '2.16.840.1.113883.10.20.22.4.85'
+            case qrda_version
```

should these `case` statements be more like:
```
when 'r3_1'
  ...
when 'r3'
  ...
when 'r2'
  ...
else
  < some sort of exception for an unsupported version >
```

As written, we default to `r2`. If we were to update to `r4` in the future but forget to update this `case` statement, we would fall through and get the `r2` template. This also fails silently if someone tries to pass in an unsupported version.